fili-core/src/test/groovy/com/yahoo/bard/webservice/web/apirequest/ApiRequestImplSpec.groovy

```diff
@@ -52,7 +50,7 @@ class ApiRequestImplSpec extends Specification {
 
         then: "we get a BadApiRequestException"
         BadApiRequestException exception = thrown()
-        exception.message == TABLE_GRANULARITY_MISMATCH.logFormat(granularity, nonExistingTableName)
+        exception.message == ErrorMessageFormat.TABLE_UNDEFINED.logFormat(nonExistingTableName)
```

Comment:
Because of the change from `TABLE_GRANULARITY_MISMATCH` to `TABLE_UNDEFINED` mentioned above